DBM-Icecrown/FrostwingHalls/Valithria.lua

```diff
@@ -130,9 +130,9 @@ function mod:OnCombatStart(delay)
 	self.vb.AbomTimer = 60
 	self.vb.AbomSpawn = 0
 	self:ScheduleMethod(30-delay, "StartBlazingSkeletonTimer")
-	self:ScheduleMethod(5-delay, "StartAbomTimer")
+	self:ScheduleMethod(5-delay, "StartAbomTimer" + 1)
```

Comment:
```suggestion
	self:ScheduleMethod(5-delay, "StartAbomTimer")
```
not a timer count arg